doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+$$
+val({1,3}) = \int_{\mathbb{R}}\int_{\mathbb{R}} f(x_1, X_2, x_3, X_4)d\mathbb{P}_{X_{2}X_{4}} - \mathbb{E}_{X}(f(X))
+$$
+
+Given a coalition, $S$, that doesn't include $x_i$, then the marginal contribution of $x_i$ is given by $val(S \cup x_i) - val(S)$. Intuitively this is the difference that the feature $x_i$ would contribute if it was to join that coalition. We are interested in the marginal contribution of $x_i$ over all possible coalitions with and without $x_i$. A Shapley value for the $x_i^{th}$ feature is given by the weighted sum
+
+$$
+\psi_j = \sum_{S\subset \{1,...,p\} \setminus \{j\}} \frac{|S|!(p - |S| - 1)!}{p!}(val(S \cup x_i) - val(S))
+$$
+
+The weights convey how much you can learn from a specific coalition. Large and Small coalitions mean more learned because we've isolated more of the effect. At the same time, medium size coalitions don't supply us with as much information because there are many possible such coalitions.
+
+The main issue with the above is that there will be many possible coalitions, $2^M$ to be precise. Hence instead of computing all of these, we use a sampling process on the space of coalitions and then estimate the Shapley values by training a linear model. Because a coalition is a set of players/features that are contributing to a prediction, we represent this as points in the space of binary codes $z' = \{z_0,...,z_m\}$ where $z_j = 1$ means that the $j^th$ feature is present in the coalition while $z_j = 0$ means it is not. To obtain the dataset on which we train this model, we first sample from this space of coalitions then compute the values of $f$ for each sample. We obtain weights for each sample using the Shapley Kernel:
+
+$$
+\pi_{x}(z') = \frac{M - 1}{\frac{M}{|z'|}|z'|(M - |z'|)}
+$$
+
+Once we have the data points, the values of $f$ for each data point, and the sample weights, we have everything we need to train a linear model. The paper shows that the coefficients of this linear model are the Shapley values.
+
+There is some nuance to how we compute the value of a model given a specific coalition, as most models aren't built to accept input with arbitrary missing values. If $D$ is the underlying distribution the samples are drawn from, then ideally, we would use the conditional expectation:
+
+$$
+f(S) = \mathbb{E}_{D}[f(x)|x_S]
+$$
+
+Computing this value is very difficult. Instead, we can approximate the above using the interventional conditional expectation, which is defined as:
+
+$$
+f(S) = \mathbb{E}_{D}[f(x)|do(x_S)]
+$$
+
+The $do$ operator here fixes the values of the features in $S$ and samples the remaining $\bar{S}$ feature values from the data. A Downside of interfering in the distribution like this can mean introducing unrealistic samples if there are dependencies between the features.
```

Comment:
Let's discuss how we can compress this information or present it in a shorter, more informal way.

Reply:
I actually like this exposition, but if we're attempting to shorten the overall document and make it more accessible we may need to rethink.

Reply:
To calibrate how does the following scan to you?


Kernel SHAP is a method of computing the Shapley values for a model around an instance. Shapley values are a game-theoretic method of assigning payout to players depending on their contribution to an overall goal.

Given any subset of features, we can ask how a feature's presence in that set contributes to the model output. We do this by computing the model output for the set with and without the specific feature. We obtain the Shapley value for that feature by considering these contributions with and without it present for all possible subsets of features. 

Two problems arise. Firstly, most models are not trained to take a variable number of input features. And secondly, the power set is prohibitively large when there are many features.

To solve the former, we sample from the interventional conditional expectation. This replaces missing features with values sampled from the training distribution. And to solve the latter, the kernel SHAP method samples on the space of subsets to obtain an estimate.

A downside of interfering in the distribution like this is introducing unrealistic samples if there are dependencies between the features.

Reply:
Sounds good to me! I would maybe add a note on the "power set" as it is introduced (some readers may not be familiar with it), and emphasise "interventional conditional expectation" to signal that it's a kind of definition (explanation follows in the next sentence already).